### PR TITLE
Add documentation for turf-aggregate

### PR DESCRIPTION
**Pull request by lyzidiamond**
PR for the first one, will commit directly to master for the others (if that's cool).

/cc @tmcw @morganherlocker

### CODE REVIEW

**Comment by tmcw** [comment]
:+1: awesome - want rpl-ize this too, so you can get the hang of that off the bat? I'll add inline notes

File: index.js [hunk 1]
```diff
@@ -16,6 +16,77 @@ operations.deviation = deviation;
...
+/**
+* Takes a set of polygons, a set of points, and an array of aggregations, then performs them. Sum, average, count, min, max, and deviation are supported.
+*
+* @module turf/aggregate
+* @param {FeatureCollection} polygonFC - a GeoJSON FeatureCollection containing Polygons
+* @param {FeatureCollection} pointFC - a FeatureColelction containing Points
+* @param {Array} aggregations - an array of aggregation objects
+* @return {FeatureCollection} polys - a FeatureCollection containing polygons with specified aggregation statistics as attributes
+* @example
+* var aggregate = require('turf-aggregate')
```
Comment (tmcw) [inline index.js]: remove these four lines with `require()` calls - we can count on `turf` being available

File: index.js [hunk 2]
```diff
@@ -16,6 +16,77 @@ operations.deviation = deviation;
...
+* var point = require('turf-point')
+* var polygon = require('turf-polygon')
+*var featurecollection =  require('turf-featurecollection')
+*
+* var poly1 = polygon([[[0,0],[10,0],[10,10],[0,10]]])
```
Comment (tmcw) [inline index.js]: rewrite `polygon` to `turf.polygon` and the same with `point` and `featurecollection` calls

Also this polygon is invalid: should be:

``` js
var poly1 = turf.polygon([[[0,0],[10,0],[10,10],[0,10],[0,0]]])
```

File: index.js [hunk 3]
```diff
@@ -16,6 +16,77 @@ operations.deviation = deviation;
...
+*   {
+*     aggregation: 'count',
+*     inField: '',
+*     outField: 'point_count'
+*   }
+* ]
+* 
+* var polys = aggregate(polyFC, ptFC, aggregations)
+* 
+* console.log(polys)
```
Comment (tmcw) [inline index.js]: change this to `//=polys`

**Comment by lyzidiamond** [comment]
Should be ready to go but could use a once-over.

**Comment by tmcw** [comment]
:+1: merge it!

**Comment by lyzidiamond** [comment]
Hooray! Onto the next one. :goat: :egg: :vhs:

**Pull request merged**